Steps4Impact/NotificationsV2/NotificationsViewController.swift

```diff
@@ -30,41 +30,124 @@
 import UIKit
 
 class NotificationsViewController: TableViewController {
+  private lazy var plistURL: URL? = {
+    do {
+      var documentURL = try FileManager.default.url(
+        for: .documentDirectory, in: .userDomainMask,
+        appropriateFor: nil, create: false)
+      return documentURL.appendingPathComponent("notifications.plist")
```

Comment:
@samisuteria thoughts on this?

Reply:
document directory is fine. make "notifications.plist" a constant.

Reply:
@samisuteria where do we keep constants? Should I just create `let` within the view controller?